knora/dsplib/utils/xml_upload.py

```diff
@@ -353,12 +367,18 @@ def xml_upload(input_file: str, server: str, user: str, password: str, imgdir: s
         stashed_xml_texts = dict()
         stashed_resptr_props = dict()
 
+    preparation_time = datetime.now() - preparation_start
+    preparation_time_microsec = preparation_time.seconds * 1_000_000 + preparation_time.microseconds
```

Comment:
I generally prefer "duration" over "time" when referring to a time delta, but that's cosmetics